requirements/test.txt

```diff
@@ -90,7 +90,7 @@ sqlparse==0.4.2
     # via
     #   -r requirements/base.txt
     #   django
-testfixtures==6.18.5
+testfixtures==7.0.0
```

Comment:
Not any major incompatible change for now in the version [7.0.0](https://github.com/simplistix/testfixtures/blob/master/CHANGELOG.rst#700-6-jul-2022)